### PR TITLE
Exposing coverage stats on pipeline

### PR DESCRIPTION
**Pull request by cat101**

### CODE REVIEW

File: landingai/common.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ class Prediction(BaseModel):
     score: float
     """The confidence score of this prediction."""
 
+    def num_predicted_pixels(self) -> int:
```
Comment (AsiaCao) [inline landingai/common.py:37]: for consistency between child classes, i think we need the property decorator
```suggestion
    @property
    def num_predicted_pixels(self) -> int:
```

**Review by AsiaCao — APPROVED** [review]

File: landingai/common.py (hunk 1 above)
Comment (cat101) [reply landingai/common.py:37]: done

File: examples/post-processings/coral/sample_images/1196.png
Comment (AsiaCao) [inline examples/post-processings/coral/sample_images/1196.png:1]: unintended change?

Comment (cat101) [reply examples/post-processings/coral/sample_images/1196.png:1]: No... why?? I'm using some sample images...hopefully we will get real coral images soon... if not this good for now

**Pull request merged**